packages/atomic/src/components/search/result-template-components/atomic-result-image/atomic-result-image.tsx

```diff
@@ -36,27 +41,50 @@ export class AtomicResultImage implements InitializableComponent {
     return Array.isArray(value) ? value[0] : value;
   }
 
+  private handleImageError() {
+    let message =
+      'The image url is not valid. You might want to add a "fallback" property.';
+
+    if (this.fallback) {
+      message = 'The image url is not valid or could not be loaded.';
+      this.host.querySelector('img')!.src = this.fallback;
+    }
+    this.bindings.engine.logger.error(message, this.host);
+  }
+
   public render() {
-    const url = this.url;
+    let url = this.url;
 
     if (!url) {
-      this.host.remove();
-      return;
+      if (this.fallback) {
+        this.bindings.engine.logger.error(
+          `"${this.field}" is missing. The fallback is being used. Please review the indexing.`,
+          this.host
+        );
+        url = this.fallback;
+      } else {
+        this.bindings.engine.logger.error(
+          `"${this.field}" is missing. Please review the indexing and add a "fallback" property.`,
+          this.host
+        );
+        this.host.remove();
+        return;
+      }
     }
 
-    if (typeof url !== 'string') {
+    if (url && typeof url !== 'string') {
       this.bindings.engine.logger.error(
         `Expected "${this.field}" to be a text field.`,
```

Comment:
Don't we want to use the fallback in this case?

Reply:
Yes you are right, I changed my code to cover this case as well.